Gemfile

```diff
@@ -2,6 +2,11 @@ source "https://rubygems.org"
 gemspec :name => "jekyll"
 
 gem "rake", "~> 11.0"
+
+if RUBY_ENGINE == 'ruby' && RUBY_VERSION < '2.2.2'
+  gem "activesupport", "< 5", :groups => [:test_legacy, :site]
```

Comment:
Thanks for the review! It's used in [html-pipeline](https://rubygems.org/gems/html-pipeline) which is a dependency of both [jemoji](https://rubygems.org/gems/jemoji) and [jekyll-mentions](https://rubygems.org/gems/jekyll-mentions). Will try to get it into one line and change the version requirement.